scripts/survival_migration/survivalDataMigration.py

```diff
@@ -0,0 +1,178 @@
+import csv
+import pandas as pd
+import numpy as np
+import os
+import sys
+import argparse
+import shutil
+
+PARENT_PARSER_NAME = 'survival data migration tool'
+NULL_VALUES = ["[not applicable]", "[not available]", "[pending]", "[discrepancy]", "[completed]", "[null]", "", "na"]
+ZERO_MAPPING_FILE_PATH = os.getcwd() + '/survivalStatusVocabulariesMappingToZero.txt'
```

Comment:
I would combine this into one file with two columns.  If there is a many to one mapping in either direction, I would allow for multiple values in a field.

Reply:
@n1zea144 do we still allow user to provide a single vocabulary that will map to 0 or 1? Or should I put an empty string for not provided value?